package.json

```diff
@@ -32,27 +39,21 @@
   "files": [
     "dist"
   ],
-  "lint-staged": {
-    "./src/**/*.{ts,js,jsx,tsx}": [
-      "eslint --ignore-path .gitignore --fix"
-    ],
-    "*": "prettier --write"
-  },
   "engines": {
     "node": ">=14.0.0"
   },
   "devDependencies": {
-    "@storybook/addon-a11y": "^7.0.2",
-    "@storybook/addon-essentials": "^7.0.2",
-    "@storybook/addon-interactions": "^7.0.2",
-    "@storybook/addon-mdx-gfm": "7.0.2",
-    "@storybook/blocks": "^7.0.2",
-    "@storybook/design-system": "7.15.10",
+    "@storybook/addon-a11y": "^7.0.18",
+    "@storybook/addon-essentials": "^7.0.18",
+    "@storybook/addon-interactions": "^7.0.18",
+    "@storybook/addon-mdx-gfm": "7.0.18",
+    "@storybook/blocks": "^7.0.18",
+    "@storybook/design-system": "7.15.11",
```

Comment:
Is there any value in pegging this to [what's in Chromatic](https://github.com/chromaui/chromatic/blob/main/package.json#L128), which is currently `7.15.9` to avoid [styling clashes that were cropping up](https://chromaticqa.slack.com/archives/C03047TQEAH/p1684536663512399).

Reply:
(That's not rhetorical—I'm not sure if it's important or necessary for us to keep this library's dependencies in sync at that level.)

Reply:
So, `@storybook/design-system` is only used for `loadFontsForStorybook` in the Storybook config. I'll move that logic here and remove ds as a dependency. Does that sound okay to you?

Reply:
Sounds good to me! 🙂

Reply:
I believe the clashes were coming from `@storybook/theming` being a different version. I fixed tetra so that it no longer packages `@storybook/theming` in the build. But maybe I'll pin that one.

Reply:
Ah, I misremembered. Thanks!